internal/configs/sharding/mainnet.go

```diff
@@ -61,7 +61,7 @@ func (ms mainnetSchedule) VdfDifficulty() int {
 
 var mainnetReshardingEpoch = []*big.Int{big.NewInt(0), big.NewInt(mainnetV1Epoch)}
 var mainnetV0 = MustNewInstance(4, 150, 112, genesis.HarmonyAccounts, genesis.FoundationalNodeAccounts, mainnetReshardingEpoch)
-var mainnetV1 = MustNewInstance(4, 152, 112, genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1, mainnetReshardingEpoch)
+var mainnetV1 = MustNewInstance(4, 160, 112, genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1, mainnetReshardingEpoch)
```

Comment:
it won't work here.  you need to add another epoch transition with a new mainnetv2 configuration.  All the past instances shouldn't be changed at all.